### ISSUE TITLE
states missing 2020 results

### ISSUE THREAD

**Issue by rootsmusic** [alarm-redist/census-2020#11]
VEST has 2020 election results from [PA](https://dataverse.harvard.edu/file.xhtml?fileId=5420076&version=25.0) and [VA](https://dataverse.harvard.edu/file.xhtml?fileId=5371929&version=25.0), but they're missing in this repo.

**Comment by christopherkenny** [comment]
Yes, those have both been added since the last data update on this repo. We run the script to retabulate the data to 2020 VTDs manually. I'll be able to update sometime in the coming week or so.

**Comment by rootsmusic** [comment]
Thanks!

**Comment by rootsmusic** [comment]
@christopherkenny And [MO](https://dataverse.harvard.edu/file.xhtml?fileId=5007850&version=35.0) too.
